packages/react-native-reanimated/src/logger/logger.ts

```diff
@@ -39,7 +39,7 @@ function logToConsole(data: LogData) {
 export const DEFAULT_LOGGER_CONFIG: LoggerConfigInternal = {
   logFunction: logToConsole,
   level: LogLevel.warn,
-  strict: true,
+  strict: false,
```

Comment:
I decided to change this to `false` as the invalid raed/write warning was reported to be too annoying and it might be better to keep it disabled by default.

Reply:
I saw a [tweet](https://x.com/hirbod_dev/status/1849491666610077954?s=46&t=rZ58uY0MGSMkjVkZUDCj3g) in which people complain about these warnings so I though that it may be too annoying. Maybe we shouldn't bother and we should encourage people to fix invalid shared value access issues on their side, though. I don't know which approach we prefer.

Reply:
We agreed to leave this warning enabled by default.